src/renderer/components/dock/upgrade-chart/clear-upgrade-chart-tab-data.injectable.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable, lifecycleEnum } from "@ogre-tools/injectable";
+import { bind } from "../../../utils";
+import type { TabId } from "../dock/store";
+import type { UpgradeChartTabStore } from "./store";
+import upgradeChartTabStoreInjectable from "./store.injectable";
+
+interface Dependencies {
+  upgradeChartTabStore: UpgradeChartTabStore;
+}
+
+function clearUpgradeChartTabData({ upgradeChartTabStore }: Dependencies, tabId: TabId) {
+  upgradeChartTabStore.clearData(tabId);
+}
+
+const clearUpgradeChartTabDataInjectable = getInjectable({
+  instantiate: (di) => bind(clearUpgradeChartTabData, null, {
```

Comment:
This bind pattern is just complexity which shouldn't be added. Configuration (e.g. dependencies) shouldn't be mixed with arguments. Arrow function would be much simpler.

Reply:
Inline `clearUpgradeChartTabData`